requirements-dev.txt

```diff
@@ -8,6 +8,7 @@ wheel
 jupyter
 setuptools
 twine
+google-re2
```

Comment:
This should be moved to requirements.txt with `google-re2[text]`. See https://pip.pypa.io/en/stable/reference/requirement-specifiers/#overview.

Reply:
If I'm not mistaken writing `google-re2[text]` in the requirements.txt file just means we request the google-re2 package with a nonexistent text option (we want the opposite).